scripts/real_scientific_search.py

```
@@ -0,0 +1,394 @@
+#!/usr/bin/env python3
+"""
+🔬 Поиск реальных научных источников для почвенных датчиков
+Системный и рекурсивный поиск подтвержденных научных работ
+"""
+
+import json
+import time
+from typing import Dict, List, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)